etc/kayobe/bifrost.yml

```
@@ -135,7 +135,7 @@
 # Ironic Python Agent (IPA) configuration.
 
 # URL of Ironic Python Agent (IPA) kernel image.
-#kolla_bifrost_ipa_kernel_upstream_url:
+kolla_bifrost_ipa_kernel_upstream_url: "{% if stackhpc_ipa_image_bifrost_enabled | bool %}{{ stackhpc_ipa_image_url }}/ipa.kernel{% endif %}"
```

I think we use this style of conditional more in SKC
```suggestion
kolla_bifrost_ipa_kernel_upstream_url: "{{ (stackhpc_ipa_image_url + '/ipa.kernel') if stackhpc_ipa_image_bifrost_enabled | bool }}"
```